lib/driver.js

```diff
@@ -758,6 +758,17 @@ class XCUITestDriver extends BaseDriver {
       log.info(`Forcing shouldUseSingletonTestManager capability value to true, because of known XCTest issues under 9.3 platform version`);
       shouldUseTestManagerForVisibilityDetection = true;
     }
+    if (util.hasValue(this.opts.language)) {
+      args.push('-AppleLanguages');
```

Comment:
it is possible pass multiple arguments to `push` method

Reply:
Thanks. Will do that